src/main/java/frc/robot/commands/RaiseMailbox.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+
```

Comment:
JavaDoc

I will note that commands are one of the few exceptions to our "JavaDoc everything public!" rule—we only document the actual command class itself, and not any of its methods, since all of a command's methods are defined by WPILib (and will actually inherit JavaDoc from the Command superclass)